dune

```diff
@@ -21,8 +21,8 @@
    %{targets}
    (run ocamlformat --help=plain))))
 
-(alias
- (name runtest)
+(rule
+ (alias runtest)
```

Comment:
Is this required by the upgrade to 2.2 ?

Reply:
yes, `(alias)` does not exist anymore in `(lang dune 2.2)`.

Reply:
It is used in `test/passing/dune.inc` (of the same form) and the tests are running fine. Why ?

Reply:
I missed `test/passing/dune-project`. Looks good